crates/ctranslate2-bindings/build.rs

```diff
@@ -54,26 +54,28 @@ fn link_static() -> PathBuf {
             .define("WITH_MKL", "OFF")
             .define("OPENMP_RUNTIME", "NONE");
 
-        if cfg!(target_arch = "aarch64") && check_jetson() {
-            config
-               .define("WITH_CUDA", "ON")
-               .define("WITH_CUDNN", "ON")
-               .cxxflag("-mcpu=native")
-        } else if cfg!(target_feature = "sse4.1") {
-            config
-                .define("WITH_OPENBLAS", "ON")
-                .cxxflag("-msse4.1")
+        if cfg!(target_feature = "sse4.1") {
+            config.cxxflag("-msse4.1");
+        }
+        
+        if cfg!(feature = "link_static_cuda") {
+            config.define("WITH_CUDA", "ON");
+            
+            if cfg!(target_arch = "aarch64") && check_jetson() {
```

Comment:
```suggestion
            if cfg!(target_arch = "aarch64") {
```

check_jetson is no longer needed as you'll need to turn on link_static_cuda manually```

Reply:
OK, got it.